patches/getting-started.diff

```diff
@@ -19,32 +19,89 @@ Index: code-server/lib/vscode/src/vs/workbench/contrib/welcomeGettingStarted/bro
  		);
  
++		let gettingStartedCoder: HTMLElement = $('.header', {});
 +		if (this.contextService.contextMatchesRules(IsEnabledCoderGettingStarted)) {
-+			header = $('.header', {},
-+			   $('h1.product-name.caption', {}, this.productService.nameLong),
-+			   $('p.subtitle.description.coder', {},
-+				   "Using code-server on a team?",
-+			   ),
-+			   $('p.subtitle.description.coder-coder', {},
-+				   "Check out: ",
-+				   $('a', { href: "https://github.com/coder/coder" }, "coder/coder")
-+			   ),
-+		   );
++			gettingStartedCoder = $('.gettingStartedCategory', {},
++				$('h2', {
++					style: 'margin-bottom: 12px',
++				}, 'Next Up'),
++				$('a', {
++					href: 'https://cdr.co/code-server-to-coder',
++					target: '_blank',
++				}, 
++					$('button', {
++						style: [
++							'padding: 10px 16px	',
++							'border-radius: 4px',
++							'background: linear-gradient(94.04deg, #7934DA 0%, #4D52E0 101.2%)',
++							'color: white',
++							'overflow: hidden',
++							'margin-right: 14px',
++						].join(';'),
++					},
++					$('h3', {
++						style: [
++							'margin: 0px 0px 6px',
++							'font-weight: 500',
++						].join(';'),
++					}, 'Deploy code-server for your team'),
++					$('p', {
++						style: [
++							'margin: 0',
++							'font-size: 13px',
++							'color: #B3BFD7',
++						].join(';'),
++					}, 'Provision remote development environments on your infrastructure with Coder.'),
++					$('p', {
++						style: [
++							'margin: 0',
++							'margin-top: 8px',
++							'font-size: 13px',
++							'display: flex',
++							'align-items: center',
++						].join(';'),
++					}, 'Get started ', $('span', {
++						class: Codicon.arrowRight.classNames,
++						style: [
++							'color: white',
++							'margin-left: 8px',
++						].join(';'),
++					})),
++					$('img', {
++						src: './_static/src/browser/media/templates.png',
++						style: [
++							'margin-bottom: -65px',
++						].join(';'),
++					}),
++					),
++				),
++			);
 +	   }
 +
  
  		const leftColumn = $('.categories-column.categories-column-left', {},);
  		const rightColumn = $('.categories-column.categories-column-right', {},);
+@@ -775,13 +834,13 @@ export class GettingStartedPage extends
+ 		const layoutLists = () => {
+ 			if (gettingStartedList.itemCount) {
+ 				this.container.classList.remove('noWalkthroughs');
+-				reset(leftColumn, startList.getDomElement(), recentList.getDomElement());
+-				reset(rightColumn, gettingStartedList.getDomElement());
++				reset(leftColumn, startList.getDomElement(), recentList.getDomElement(), gettingStartedList.getDomElement());
++				reset(rightColumn, gettingStartedCoder);
+ 				recentList.setLimit(5);
+ 			}
+ 			else {
+ 				this.container.classList.add('noWalkthroughs');
+-				reset(leftColumn, startList.getDomElement());
++				reset(leftColumn, startList.getDomElement(), gettingStartedCoder);
```

Comment:
Should we gate these column changes behind
```typescript
if (this.contextService.contextMatchesRules(IsEnabledCoderGettingStarted))
```
so they don't affect users who disable this?

Reply:
Good point!

Reply:
Fixed!